src/NUnitFramework/nunitlite/Options.cs

```diff
@@ -539,23 +539,6 @@ public override void GetObjectData (SerializationInfo info, StreamingContext con
 
     public class OptionSet : KeyedCollection<string, Option>
     {
-#if !PORTABLE
```

Comment:
We never use the localizer, so easier to easier to just use the PORTABLE null-op version for everything.